docs/topics/azure-api-throttling.md

```diff
@@ -0,0 +1,550 @@
+Jun 24 18:32:48 k8s-master-31453872-2 kubelet[73220]: I0624 18:32:48.226853   73220 kubelet_pods.go:1090] Killing unwanted pod "kube-controller-manager-k8s-master-31453872-2"
+Jun 24 18:32:48 k8s-master-31453872-2 kubelet[73220]: I0624 18:32:48.230587   73220 kuberuntime_container.go:581] Killing container "docker://84f75fc07d76a4c42a3f27c5c30967e57a7dc863a04c43ec7ea7163158bc302a
+Jun 24 18:32:48 k8s-master-31453872-2 kubelet[73220]: I0624 18:32:48.393892   73220 provider.go:124] Refreshing cache for provider: *credentialprovider.defaultDockerConfigProvider
+Jun 24 18:32:48 k8s-master-31453872-2 kubelet[73220]: I0624 18:32:48.394917   73220 provider.go:124] Refreshing cache for provider: *azure.acrProvider
+Jun 24 18:32:48 k8s-master-31453872-2 kubelet[73220]: I0624 18:32:48.478356   73220 kubelet.go:1933] SyncLoop (PLEG): "kube-controller-manager-k8s-master-31453872-2_kube-system(6c557ed8279f08082ce8545b701ff
+Jun 24 18:32:49 k8s-master-31453872-2 kubelet[73220]: W0624 18:32:49.489357   73220 pod_container_deletor.go:75] Container "97ef204fed2c5dc700b248203c310ebce748dc03703ccf535b7288e992b9bdf7" not found in pod
+Jun 24 18:32:50 k8s-master-31453872-2 kubelet[73220]: I0624 18:32:50.221701   73220 kubelet_pods.go:1090] Killing unwanted pod "kube-controller-manager-k8s-master-31453872-2"
+Jun 24 18:32:50 k8s-master-31453872-2 kubelet[73220]: I0624 18:32:50.225143   73220 kuberuntime_container.go:581] Killing container "docker://84f75fc07d76a4c42a3f27c5c30967e57a7dc863a04c43ec7ea7163158bc302a
+azureuser@k8s-master-31453872-2:~$ exit
+logout
+Connection to k8s-master-31453872-2 closed.
+```
+
+## Verify controller-manager is running the desired, upgraded version
+
+Now, you can verify that v1.15.12 is the running `controller-manager` version:
+
+```
+azureuser@k8s-master-31453872-0:~$ for pod in $(kubectl get pods -n kube-system | grep kube-controller-manager | awk '{print $1}'); do kubectl logs $pod -n kube-system | grep "v1.15.12"; done
+I0624 18:32:18.102332       1 controllermanager.go:164] Version: v1.15.12
+I0624 18:32:45.174767       1 controllermanager.go:164] Version: v1.15.12
+I0624 18:33:05.825467       1 controllermanager.go:164] Version: v1.15.12
+```
+
+## Reload cluster-autoscaler addon
+
+If you also are running (and have updated) `cluster-autoscaler`, note that the prior version is still running:
+
+```
+azureuser@k8s-master-31453872-0:~$ for pod in $(kubectl get pods -n kube-system | grep cluster-autoscaler | awk '{print $1}'); do kubectl logs $pod -n kube-system | grep "1.15"; done
+I0624 18:32:40.458155       1 main.go:354] Cluster Autoscaler 1.15.3
+```
+
+That's because the `kube-addon-manager` and `cluster-autoscaler` containers continued running all the while, they were not affected by stopping the systemd kubelet service. We want to stop all the running `kube-addon-manager` pods to force a reload of the `/etc/kubernetes/addons` directory:

```

Comment:
```suggestion
That's because the `kube-addon-manager` and `cluster-autoscaler` containers continued running all the while: they were not affected by stopping the systemd kubelet service. We want to stop all the running `kube-addon-manager` pods to force a reload of the `/etc/kubernetes/addons` directory:
```

The document seems to go back and forth between "you" and "we." I slightly prefer the latter, but prefer consistency more. Still, it's not required to change, I'm just saying.

Reply:
Changed to "we" everywhere